pkg/controller/aerospikecluster/aerospikecluster_controller.go

```diff
@@ -223,16 +223,15 @@ func (r *ReconcileAerospikeCluster) reconcileCluster(aeroCluster *aerospikev1alp
 				needRollingRestart = true
 			}
 		}
+	} else {
```

Comment:
Do we need this? If accessControl and updateStatus failed earlier then this function should update them at the end. Or may be we can't even reach till end....need to discuss.